crates/core/src/client/constants.rs

```diff
@@ -54,6 +54,12 @@ pub mod gas {
     /// on a "first come first served" basis by the Sequencer.
     /// As a result, the priority fee is set to 0.
     pub const MAX_PRIORITY_FEE_PER_GAS: U128 = U128::ZERO;
+
+    /// The minimum gas fee for a transaction
+    ///
+    /// This minimum of 21,000 is used if the returned fee estimate is lower,
```

Comment:
may be worth adding some doc https://ethereum.stackexchange.com/a/34675/81474 for the record